mesonbuild/modules/qt.py

```diff
@@ -136,7 +134,7 @@ def gen_bins() -> T.Generator[T.Tuple[str, str], None, None]:
                 yield b, b
 
         for b, name in gen_bins():
-            if found[name].found():
+            if getattr(self, name).found():
```

Comment:
This isa  good cleanup, but I think we could do a bit of refactoring to make it cleaner:
```python
self.tools = {
    'rcc': ...,
    'uic': ...,
    'moc': ...,
    'lrelease': ..,
}
```

```python 
def gen_bins() -> T.Generator[T.Tuple[str, str], None, None]:
   for b in self.bins:
     ...

for b, name in gen_bins():
    if self.bins[name].found():
       ...
```
Which would remove the duplication of the bins list, since we'd have a single authoritative list of bins.

Reply:
Ok, done.